Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -398,6 +398,28 @@
         tags:
           - NukeOpsUplink
 
+- type: listing
+  id: UplinkDnaScramblerImplant
+  name: uplink-dna-scrambler-implanter-name
+  description: uplink-dna-scrambler-implanter-desc
+  icon: { sprite: /Textures/Mobs/Species/Human/parts.rsi, state: full }
+  productEntity: DnaScramblerImplanter
+  cost:
+    Telecrystal: 10
```

```suggestion
    Telecrystal: 4
```
The item is incredibly gimmicky and pairing with a voice mask and agent card is basically a must, which would set you back 19 TC total (3 TC for ID Card, 6 for Voice Mask, 10 for the implant)

---

It's a one time use ling power, you don't even need a voice mask for it.

---

The scrambler turns you in a completely new entity with it's own name, voice, DNA and fingerprints. The items you mentioned aren't really a necessity, you can just tell hop you fried your ID in the microwave and get a new passenger one to match your new identity.